sites/all/themes/custom/ogp_theme/sass/main.scss

```diff
@@ -28,6 +28,7 @@
 'layouts/l-news-events',
 'layouts/l-working-group',
 'layouts/l-working-group-list',
+'layouts/l-pages-list',
```

Comment:
If you already have the same view for groups, why do you need another css view?? Please reuse code.